### ISSUE TITLE
Can not dismiss all modals at once

### ISSUE THREAD

**Issue by tautvilas** [wix/react-native-controllers#36]
In case when I open a modal on top of another modal and then do:

```
this.props.navigator.dismissModal({animationType: 'none'});
this.props.navigator.dismissModal({animationType: 'none'});
```

second modal does not get closed. However this closes both modals:

```
this.props.navigator.dismissModal({animationType: 'none'});
setTimeout(() => {
  this.props.navigator.dismissModal({animationType: 'slide-down'});
})
```

If I were to use sliding animation in dismissing first modal the setTimeout would not help.

I would like to have navigator.dismissAllModals method, that animates only last modal dismissal and silently dismisses everything else too

**Comment by talkol** [comment]
This sparked an interesting discussion between @artald and myself..

It's possible that to dismiss two modals quickly isn't supported out of the box on iOS, because the modal doesn't stop being the presentedViewController until the dismiss animation is over.

If we have modal B over modal A, what we originally wanted to support was dismissModal(B) with animation and immediately dismissModal(A) without animation. Hopefully this would have gotten the wanted result of B (the topmost) animating away and revealing the original content. We'll need to play with this and see if it's possible to do.

We should also add support for `dismissAllModals`. It's probable that we'll need to use the same approach as setController animated and take a snapshot of the screen and do the animation manually.

**Comment by artald** [comment]
@talkol I added support for dismissing all presented view controller. see here: [b41e5ad](https://github.com/wix/react-native-controllers/commit/b41e5adb82585b80aada2787ad29263ad1f19cf3).

I think it's a solution we can live with for now and in this case we can close this issue (after you tell me that it works for you..)

**Comment by talkol** [comment]
Great job! released in 1.3.1 and added to readme and react-native-navigation
